.gitattributes

```diff
@@ -5,3 +5,4 @@ tests/data/plugins/browsers/firefox/places.sqlite filter=lfs diff=lfs merge=lfs
 tests/data/plugins/browsers/chrome/History.sqlite filter=lfs diff=lfs merge=lfs -text
 tests/data/plugins/browsers/edge/History.sqlite filter=lfs diff=lfs merge=lfs -text
 tests/data/plugins/browsers/chromium/History.sqlite filter=lfs diff=lfs merge=lfs -text
+tests/data/loaders/asdf/metadata.asdf filter=lfs diff=lfs merge=lfs -text
```

Comment:
Is it necessary to add an 11K file to LFS? I thought this was meant for large files?

Reply:
Maybe good to talk about what exactly the boundary is in a standup, because I'm not sure anymore either :)